java-sdk-example/README.md

```diff
@@ -16,6 +16,8 @@ This example application uses Signadot's Java SDK to create a sandbox and test t
 The test creates a sandbox in the setup phase, runs tests against it and deletes the sandbox upon completion of the tests. Heres' the command to run it:
 
 ```
+export SIGNADOT_ORG=...
 export SIGNADOT_API_KEY=...
+export SIGNADOT_CLUSTER_NAME=...
```

Comment:
Arjun had once mentioned that he finds the prefix `Signadot` (eg. for signadot org) confusing. So maybe we can go without it while you're at it?

Reply:
I am a bit worried that things like `CLUSTER_NAME` feel underspecified when we don't qualify with the `SIGNADOT_` prefix. WDYT?

Reply:
I feel `SIGNADOT_API_KEY` can remain as is as the users may be dealing with API keys for multiple things. But I'm less concerned about the other two. Thoughts, @arjun-sd ?

Reply:
I don't have a strong opinion here, so, if you think we should remove it across the board, I can make that change.